src/Asv.Avalonia.IO/Pages/Settings/Connections/PortViewModel.cs

```diff
@@ -193,12 +193,12 @@ protected void ResetChanges()
     {
         if (string.IsNullOrWhiteSpace(arg))
         {
-            return new Exception("Port name is required");
+            return new Exception(RS.PortViewModel_ValidateName_Exception_PortNameRequired);
         }
 
         if (arg.Length > 50)
         {
-            return new Exception("Port name is too long. Max length is 50 characters");
+            return new Exception(RS.PortViewModel_ValidateName_Exception_PortNameTooLong);
```

Comment:
```suggestion
return new Exception(RS.PortViewModel_ValidateName_PortNameTooLongException);
```
Выше такая же логика